### DISCUSSION TITLE
I’m not the smartest but I have some ideas for the notification

### DISCUSSION THREAD

**Discussion by TheDeac** [benawad/dogehouse#1006] [Ideas]
1.This might or might not work but maybe if you were to sign in with a phone number or an email it could send the notification to the the email or phone number
2.if there is a profile it would be really cool if you could customize it like add your own code or just add simple thing to your profile that would be really cool as well

**Comment by dk-raw** [comment]
There is no email sign up, only Twitter and GitHub. For the 2nd suggestion, open an Issue because it seems like a good one.
